app/classifier/tasks/survey/index.cjsx

```diff
@@ -84,7 +87,13 @@ module.exports = React.createClass
   handleAnnotation: (choice, answers, e) ->
     filters = JSON.parse JSON.stringify @state.filters
     value = @props.annotation.value?.slice?(0) ? []
-    value.push {choice, answers, filters}
+    # @ is undefined within the scope of findIndex
+    currentSelection = @state.selectedChoiceID
+    annotationIndex = value.findIndex (value) -> value.choice is currentSelection
```

Comment:
@eatyourgreens btw just passing `currentSelection` as it is doesn't work. There's probably a way to make good old `indexOf` work here, but to me it seems easier to just add `findIndex` to the polyfill url. Unless you show me a clean method to get `indexOf` to work.